stable/postgresql/templates/deployment.yaml

```diff
@@ -32,29 +32,35 @@ spec:
             secretKeyRef:
               name: {{ template "fullname" . }}
               key: postgres-password
+        - name: POD_IP
+          valueFrom: { fieldRef: { fieldPath: status.podIP } }
         ports:
         - name: postgresql
           containerPort: 5432
         livenessProbe:
```

Comment:
Maybe we should add a little comment here why we use the POD_IP in the probes. To prevent this being "simplified" in the future.

E.g.: $POD_IP is used here to prevent the pod from becoming ready while running the initial database setup.